index.html

```diff
@@ -58,6 +58,12 @@ <h1>Open-Source</h1>
         all.</p>
     </section>
 
+    <section>
+      <h1>User Friendly</h1>
+      <p>We have a safe and welcoming community that will guide youu through Meower! <a href="https://app.meower.org"
```

Comment:
youu

Reply:
what

Reply:
pls read

Reply:
not resolved, it says "will guide **youu**"